### DISCUSSION TITLE
dde-dock: DateTime color

### DISCUSSION THREAD

**Discussion by micazi** [linuxdeepin/developer-center#3054] [Features Request & Ideas | 特性请求 & 头脑风暴]
Date and time on the dock should change it's color based on the background to make it contrast.

**Comment by 0yanghongwei** [comment]
@micazi Current color should be quite obvious. I don't know which situation you are talking about will not be obvious. Can you take a screenshot?

**Comment by 0yanghongwei** [comment]
@micazi Hello, this requirement has not been added to the development plan at present. Thanks for your advice anyway.

**Comment by meiyixiang** [comment]
Your suggestion is more of a visual experience problem, such problems can cause global disharmony due to what the UI designer decided at design time.
